openapi/paths/credit-memos@{id}.yaml

```
@@ -34,11 +34,11 @@ put:
     - Core
   tags:
     - Credit memos
-  summary: Create or update a credit memo with predefined ID
+  summary: Create or update a credit memo with specified ID
   operationId: PutCreditMemo
   x-sdk-operation-name: update
   description: |
-    Create or update a credit memo with predefined identifier string.
+     Create or update (upsert) a credit memo with specified identifier string.
```

```suggestion
     Creates or updates (upsert) a credit memo with specified identifier string.
```

There are occurences both for `Creates or updates` and for `Create or update`, should we standardize this as well?

---

I haven’t reworked many of the descriptions yet. I thought it would be best to do a find and replace for all first, and then work through the remaining descriptions afterward